src/main/java/Frontend/CommandLineInterface.java

```diff
@@ -1,7 +1,38 @@
 package Frontend;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class CommandLineInterface {
+    private static final String[] acceptedCommands = {"-eq", "-save", "-domain", "-2d", "-3d",
```

Comment:
We likely don't need this to be static

Reply:
We need this to be static since this element 'acceptedCommands' is accessed within the static method main, intelliJ is also giving the error "Non-static field 'acceptedCommands' cannot be referenced from a static context"